src/presentation/Console.java

```diff
@@ -168,13 +170,16 @@ public static boolean promptToContinue() {
 
 			if (matchesAffirmative) {
 				isValid = true;
+				wantsToPlayAgain = true;
 			} else if (matchesNegative) {
-				isValid = true;
+				isValid = false;
+				wantsToPlayAgain = false;
+				break;
```

Comment:
Hmm, close, but not quite. You are just forcing a break from the loop. Breaking from a loop is a valid strategy I think, but in this case, we are combining using a variable for the loop logic, but we are just breaking from the loop when the logic doesn't work the way that we expect. See if you can figure this out without using a break anywhere inside of the loop...